crates/polars-plan/src/plans/ir/tree_format.rs

```diff
@@ -883,3 +883,43 @@ impl fmt::Debug for TreeFmtVisitor {
         Ok(())
     }
 }
+
+// GraphViz Output
+// Create a simple DOT graph String from TreeFmtVisitor
+
+// Adding a non-standard display trait is not the right way to do this.
+// Not sure how to cleanly communicate this function to the higher level which
+// only wants traits.
```

Comment:
As you can see, I'm still a bit new to Rust and was not quite sure how to do this cleanly. Suggestions appreciated!

Reply:
Maybe the `TreeFmtVisitor` struct could hold a boolean flag that's used to switch the behaviour of `fmt::Display::fmt` between dot or regular text formatting, and we could create a new `into_dot_tree_formatter` method that's similar to `into_tree_formatter` but sets that flag.